src/timer.rs

```diff
@@ -1,14 +1,18 @@
+//! Async timer driver. Can be used for (non-busy)  sleeping.
+
 use crate::callback::CallbackSubscription;
 use crate::callback::SubscribableCallback;
 use crate::futures;
 use crate::result::OtherError;
+use crate::result::TockError;
 use crate::result::TockResult;
 use crate::syscalls;
 use core::cell::Cell;
 use core::isize;
 use core::ops::{Add, AddAssign, Sub};
 
 const DRIVER_NUMBER: usize = 0x00000;
+const EALREADY: isize = -3;
```

Comment:
Shouldn't error codes be part of the `TockError` (or `result`) module? This code isn't specific to timer.

Reply:
solved